.github/workflows/lint_test_build.yml

```diff
@@ -18,9 +18,32 @@ jobs:
       - uses: ./.github/workflows/rafiki/env-setup
       - run: pnpm checks
 
-  backend:
+  graphql:
     runs-on: ubuntu-latest
     needs: checkout
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/workflows/rafiki/env-setup
+
+      - name: generate backend graphql
+        run: pnpm --filter backend generate
+      - name: generate auth graphql
+        run: pnpm --filter auth generate
+
+      - name: verify changed files
+        uses: tj-actions/verify-changed-files@v13
+        id: verify-changed-files
+        with:
+          files: |
+            **/generated/graphql.*
```

Comment:
:heavy_check_mark: 
- https://github.com/interledger/rafiki/pull/1150

but I'm surprised this took ~90 seconds

Reply:
Me too. On their website it said it only takes ~10 seconds

Reply:
We should talk to TJ